src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] All the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Kindly do so.

Reply:
Thanks for pointing out these. I have deleted the files created through CRA.